### PR TITLE
Automatic Browser Open

### LINKED ISSUE #25: Open the browser automatically
Implement a function open a browser automatically when the local server is started.

### PR DESCRIPTION
**Pull request by king-11**
Add feature to open browser automatically.

Fixes #25

### CODE REVIEW

**Comment by king-11** [comment]
@yusukebe can you please take a look

**Comment by yusukebe** [comment]
Hi @king-11 

Great! One thing. How about adding an option to disable automatic browser opening. Such as `--disable-auto-open`.

**Comment by king-11** [comment]
Sounds good to me I will make changes for it

**Comment by king-11** [comment]
@yusukebe I have added in the flag please take a look

**Review by yusukebe — APPROVED** [review]
Good 👍

File: server.go [hunk 1]
```diff
@@ -76,6 +77,9 @@ func (server *Server) Serve(param ServerParam) {
 		fmt.Printf("master slides can be found at http://*:%d/master/\n", port)
 	}
 
+	if !param.DisableAutoOpen {
+		go openBrowser(port)
```
Comment (yusukebe) [inline server.go:81]: @king-11 

Lint throwing an error:
https://github.com/yusukebe/revealgo/runs/5000226417?check_suite_focus=true#step:4:69

Please check it.

**Review by yusukebe — CHANGES_REQUESTED** [review]

File: server.go (hunk 1 above)
Comment (king-11) [reply server.go:81]: Made the changes

**Comment by yusukebe** [comment]
@king-11 
Thank you! I merged it.

**Pull request merged**